BookEcommerce.Models/DAL/Repositories/SendMailRepository.cs

```diff
@@ -58,6 +65,8 @@ public async Task SendMailAsync(MailSendingViewModel SendMailDTO)
             }
             catch (Exception ex)
             {
+                logger.LogError(ex.Message);
+                logger.LogError(ex.StackTrace);
                 throw new Exception("fail to send mail");
```

Comment:
add 1 log for 1 error logger.LogError($"Send mail failed because {ex.Message}. Detail {ex.StackTrace});

Reply:
resolved